src/test/java/athena/StorageTest.java

```diff
@@ -59,19 +58,26 @@ private boolean areFilesSame(String file1, String file2) {
     @Test
     void loadTaskListData_saveFileFound_createTaskList() {
         Ui ui = new Ui();
-        Storage storage = new Storage("src/test/java/athena/StorageTestAnswer1.csv", ui);
-        TaskList taskList;
-        taskList = storage.loadTaskListData();
+        Storage storage = new Storage("src/test/java/athena/StorageTestAnswer1.csv");
+        TaskList taskList = null;
+        try {
+            taskList = storage.loadTaskListData();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
         TaskList tester = TestSetup.getTestTaskList();
         assertTrue(tester.equals(taskList));
     }
 
     @Test
     void loadTaskListData_commaInTaskAttribute_commaIsReplaced() {
-        Ui ui = new Ui();
-        Storage storage = new Storage("src/test/java/athena/StorageTestAnswer2.csv", ui);
-        TaskList taskList;
-        taskList = storage.loadTaskListData();
+        Storage storage = new Storage("src/test/java/athena/StorageTestAnswer2.csv");
+        TaskList taskList = null;
+        try {
+            taskList = storage.loadTaskListData();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
I believe the test should fail if the this part fails, so we shouldn't catch and handle this exception.